docs/language-server/protocol-language-server.md

```diff
@@ -195,7 +195,6 @@ transport formats, please look [here](./protocol-architecture).
   - [`EmptyStackError`](#emptystackerror)
   - [`InvalidStackItemError`](#invalidstackitemerror)
   - [`ModuleNotFoundError`](#modulenotfounderror)
-  - [`ModuleNotFoundForExpressionError`](#modulenotfoundforexpressionerror)
```

Comment:
Wasn't the error just added recently? ;-)

Reply:
:sweat_smile: